_posts/2022-03-22-vscode-quarkus-1.10.0-qute.adoc

```diff
@@ -0,0 +1,191 @@
+---
+layout: post
+title: 'Qute Templating Engine Support - Now Available for Quarkus Tools for Visual Studio Code'
+synopsis: Quarkus Tools for Visual Studio Code version 1.10.0 introduces its brand new Qute Templating Engine support.
+date: 2022-04-04
```

Comment:
You should rename the file to be consistent with your publication date.

Then if everything is ready for you, we can merge and it will be published on due date.

Reply:
Renamed to `2022-04-04-vscode-quarkus-1.10.0-qute.adoc`